core/common/src/test/java/alluxio/ConfigurationRule.java

```diff
@@ -54,6 +54,7 @@ public void evaluate() throws Throwable {
         try {
           statement.evaluate();
         } finally {
+          Configuration.defaultInit();
```

Comment:
@apc999 I don't think we want this. What if someone used one class-level configuration rule and a couple method-level configuration rules? This would lose the class-level configuration rule after the first method-level rule finished.